### DISCUSSION TITLE
Why is this crashing?

### DISCUSSION THREAD

**Discussion by giftedl** [FabricMC/fabric-api#3014] [Mod Dev Support] [answered]
I cloned [Recode](https://www.github.com/homchom/recode), and i tryed building it. And i got a error, and i have absolutely no idea how to fix. I dont really know how JDK fatal errors work, so i just commented it here.

I did the following:
- [ ] fabric works through the minecraft launcher, just it doesnt work when building through intellj
- [ ] go through every single JDK i have
- [ ] restart my computer 
- [ ] restart intellj
- [ ] switch off and on the [intellj build and run using & run tests using options](https://fabricmc.net/wiki/tutorial:setup#manual_steps)

Error: https://gist.github.com/DeveloLongScript/5b3adba00f53e657cdd622aca5c15194

If you here and planning to answer, thanks :heart:

**Comment by Juuxel** [comment]
JDK 19 is too new for the version of LWJGL used by the game. You need to either use JDK 17 or upgrade the LWJGL version locally.

**Reply by giftedl** [reply]
Welp. Does this work?
![idea64_mHIZOvxdHq](https://user-images.githubusercontent.com/52332868/232623027-9c6c658f-c61a-4d23-bb3a-b7f8c82a8726.png)
because this one crashed

**Reply by giftedl** [reply]
Also, my jdk is:
![idea64_Jgp2XYD0pm](https://user-images.githubusercontent.com/52332868/232639922-12be6dce-f97c-4bac-9024-4ccea103b712.png)

**Comment by giftedl** [comment]
As a side comment, I looked at the dll mentioned in the crash:

> `# C  [atio6axx.dll+0x192b60]`

and as seen here, this might be [a issue with the AMD video card I have](https://gaming.stackexchange.com/questions/302863/minecraft-crashing-before-during-start-up-help).
I'm not closing this quite yet because I don't quite know.

**Comment by giftedl** [comment, marked as answer]
Yay, i just had to reinstall the AMD drivers.

Thanks for the help @Juuxel
